index.html

```diff
@@ -491,9 +491,18 @@ <h2>Syntax Tokens and Keywords</h2>
         language of a JSON-LD document. This keyword is described in
         <a class="sectionRef" href="#string-internationalization"></a>.</dd>
       <dt><code>@type</code></dt>
-      <dd>Used to set the data type of a <a>node</a> or
-        <a>typed value</a>. This keyword is described in
-        <a class="sectionRef" href="#typed-values"></a>.</dd>
+      <dd>Used to set the type of a <a>node</a> or the datatype of a <a>typed value</a>.
+        This keyword is described further in <a class="sectionRef" href="#specifying-the-type"></a>
+        and <a class="sectionRef" href="#typed-values"></a>.
+        <div class="note">The notion of type can be confusing, as both
```

Comment:
I wouldn't start by saying that this can be confusing. It's confusing to people that have deep knowledge, but for beginners I don't expect it is. 

Things have types. Nodes have types. Values have types. You use `@type` to express that. That's not confusing. 

The confusing part is probably the *chording* of `@type` with other things like `@value` and a node to get the expression you want... but we really only do that in one place that only experts tend to see.

In any case, I'd reword to something like:

The use of <code>@type</code> to define a type for both <a>node objects</a> and <a>value objects</a>        addresses the basic need to type data, be it a literal value or a more complicated <a>resource</a>. Experts may find the overloaded use of the `@type` keyword for both purposes concerning, but should note that Web developer usage of this feature over multiple years has not resulted in its misuse due to the far less frequent use of `@type` to express typed literal values.

Reply:
But, the fact is, for many it seems to be a confusing issue. But, I like your wording better!